fe/src/features/students/components/DeleteStudentDialog.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <q-card style="min-width: 350px">
+    <q-card-section horizontal>
+      <q-card-section class="col-3 flex flex-center">
+        <q-icon
+          name="warning"
+          class="text-red"
+          size="80px"
+        />
+      </q-card-section>
+      <q-card-section class="q-pt-xs">
+        <div class="text-h6 q-mt-sm q-mb-xs">
+          Are you sure you want to delete "{{ name }}"?
+        </div>
```

Comment:
I would add some more text to the dialog instead of only a title for example:
```suggestion
        <div class="text-h6 q-mt-sm q-mb-xs">
          Are you sure you want to delete "{{ name }}"?
        </div>
        <div class="text text-grey">
          This student will be permanently deleted. 
          You cannot  undo this action.
        </div>
```